packages/eas-cli/src/submissions/android/ServiceAccountSource.ts

```diff
@@ -80,3 +116,38 @@ async function askForServiceAccountPathAsync(): Promise<string> {
   });
   return filePath;
 }
+
+async function confirmDetectedPathAsync(path: string): Promise<boolean> {
+  Log.log(
+    `A valid Google Service Account JSON key has been found in \n   ${chalk.underline(path)}`
```

Comment:
we are only checking one field of that object, saying that we found valid object is a bit to much